OSXDemo/OCTFriendsViewController.m

```diff
@@ -0,0 +1,307 @@
+//
+//  OCTFriendsViewController.m
+//  objcTox
+//
+//  Created by Chuong Vu on 12/12/15.
+//  Copyright © 2015 dvor. All rights reserved.
+//
+
+#import "OCTFriendsViewController.h"
+#import "RBQFetchedResultsController.h"
+#import "OCTFriend.h"
+#import "OCTFriendRequest.h"
+#import "OCTSubmanagerObjects.h"
+#import "OCTSubmanagerFriends.h"
+
+static NSString *const kNibName = @"OCTFriendsViewController";
+static NSString *const kCellIdent = @"cellIdent";
+
+@interface OCTFriendsViewController () <NSTableViewDataSource,
+                                        NSTableViewDelegate,
+                                        RBQFetchedResultsControllerDelegate>
+
+@property (strong, nonatomic) OCTManager *manager;
+@property (strong, nonatomic) RBQFetchedResultsController *friendResultsController;
+@property (strong, nonatomic) RBQFetchedResultsController *friendRequestResultsController;
+
+@property (weak) IBOutlet NSButton *acceptButton;
+@property (weak) IBOutlet NSButton *rejectButton;
+@property (unsafe_unretained) IBOutlet NSTextView *friendInfoTextField;
```

Comment:
It's what Xcode automatically put in there for me. I tried making it weak but it gave me a compile error about it not supporting weak references?